dms/models/mixins_thumbnail.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2017-2019 MuK IT GmbH.
+# Copyright 2020 Creu Blanca
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import base64
+import os
+
+from odoo import api, fields, models
+from odoo.modules.module import get_resource_path
+
+
+class Thumbnail(models.AbstractModel):
+
+    _name = "dms.mixins.thumbnail"
+    _description = "Thumbnail Mixin"
+
+    custom_thumbnail = fields.Image(
```

Comment:
Does it make sense to apply here new v13 `image.mixin` model?

Reply:
Because there is two images, the original one and the computed